src/storybook/stories/MessageInfoText.stories.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {View} from 'react-native';
+import {getStaticColor} from '@atb/theme/colors';
+import {MessageInfoText} from '@atb/components/message-info-text';
+
+const MessageInfoTextMeta = {
+  title: 'MessageInfoText',
+  component: MessageInfoText,
+  argTypes: {
+    theme: {
+      control: 'select',
+      options: ['light', 'dark'],
+    },
+  },
+  args: {
+    theme: 'light',
+    message: 'The message body.',
```

Comment:
Should messages normally end with a "."?
When the icon is on the right, I think it looks better without